components/Button.vue

```diff
@@ -3,14 +3,38 @@
     <div style="color: red;">
       {{ $t(message) }}
     </div>
-    <div class="paypal-button" />
+    <div v-if="disabled">
+      <div class="paypal-button disabled-overlay" />
```

Comment:
does this actually trigger paypal flow? just curious since we have now 2 buttons.

Reply:
yes, it still does 
![image](https://user-images.githubusercontent.com/17791285/133678013-2eafd7b8-2f61-45d6-a9ce-8220b12aa51f.png)

Reply:
ok great